distribution/docs/src/main/resources/content/_tables/sourcePollerRunner.adoc

```diff
@@ -0,0 +1,27 @@
+:title: Source Poller Runner
+:id: ddf.catalog.util.impl.SourcePollerRunner
+:status: published
+:type: table
+:application: ${ddf-catalog}
+:summary: Source Poller Runner configurations.
+
+.[[_ddf.catalog.util.impl.SourcePollerRunner]]Source Poller Runner
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Poll Interval (minutes)
+|pollIntervalMinutes
+|Integer
+|The interval (in minutes) at which to recheck the availability of all sources. Must be at least 1 minute.
```

Comment:
✏️ We may want to warn the reader about using a long poll interval here as well.